src/uct/api/uct.h

```diff
@@ -1144,10 +1144,15 @@ ucs_status_t uct_md_mem_dereg(uct_md_h md, uct_mem_h memh);
  * exhausted. In this case the next allocation method from the initial list will
  * be attempted.
  *
- * @param [in]     addr        If value is not NULL, then the value will be used
- *                             as a hint or requirement for allocation at
- *                             exactly that address if flags parameter contains
- *                             UCT_MD_MEM_FLAG_FIXED bit.
+ * @param [in]     addr        If addr is NULL, the underlying allocation routine
+ *                             will choose the address at which to create the mapping.
+ *                             If addr is non-NULL but UCT_MD_MEM_FLAG_FIXED is
+ *                             not set, the address will be interpreted as a hint
+ *                             as to where to establish the mapping. If addr is
+ *                             non-NULL and UCT_MD_MEM_FLAG_FIXED is set, then
+ *                             the specified address is interpreted as a requirement.
+ *                             In this case, if the mapping to the exact address
+ *                             cannot be made, the allocation request fails.
```

Comment:
one minor fix-up:  please preface all occurrences of 'addr' in the parameter description with @a to ensure that they show up as italics in the document, e.g., "If @a addr is NULL..."